pyhub-installer/.github/workflows/release.yml

```
@@ -0,0 +1,98 @@
+            suffix: ".exe"
+          - os: ubuntu-latest
+            goos: darwin
+            goarch: amd64
+            suffix: ""
+          - os: ubuntu-latest
+            goos: darwin
+            goarch: arm64
+            suffix: ""
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)